parsedatetime/__init__.py

```diff
@@ -2351,7 +2351,7 @@ def _buildOffsets(offsetDict, localeData, indexStart):
 
         self.RE_UNITS = r'''\b(?P<qty>
                                 -?
-                                (?:\d+|{numbers})\s*
+                                (?:\d+|(?:{numbers})\b)\s*
```

Comment:
This `\b` may be unnecessary. And it will breaks strings like "3days", "5hours".